### PR TITLE
GS:OGL:Shaders: Formatting

### PR DESCRIPTION
**Pull request by TellowKrinkle**
### Description of Changes
Less aggressive version of #8648

### Rationale behind Changes
Our GLSL shaders have a lot of nested ifdefs which are incredibly confusing with the default clang-format formatting

### Suggested Testing Steps
Test OpenGL things

### CODE REVIEW

**Comment by stenzek** [comment]
This is going to conflict with #8613, and in a nasty way (I removed the GS completely).

**Comment by lightningterror** [comment]
> This is going to conflict with #8613, and in a nasty way (I removed the GS completely).

I forgot that wasn't merged yet.

**Review by JordanTheToaster — APPROVED** [review]

**Review by Mrlinkwii — APPROVED** [review]

**Review by JordanTheToaster — APPROVED** [review]

File: bin/resources/shaders/opengl/convert.glsl [hunk 1]
```diff
@@ -366,68 +366,68 @@ uniform ivec2 EMOD;
...
+	rgb2yuv[1] = vec3(0.114, 0.500, -0.081);
+	rgb2yuv[2] = vec3(0.299, -0.169, 0.500);
+
+	vec3 yuv = rgb2yuv * i.gbr;
+
+	float Y = float(0xDB)/255.0f * yuv.x + float(0x10)/255.0f;
+	float Cr = float(0xE0)/255.0f * yuv.y + float(0x80)/255.0f;
+	float Cb = float(0xE0)/255.0f * yuv.z + float(0x80)/255.0f;
+
+	switch(EMOD.x) {
```
Comment (lightningterror) [inline bin/resources/shaders/opengl/convert.glsl:383]: The brackets should begin on new row, same goes for if statements I saw some.

**Comment by lightningterror** [comment]
I'll go ahead and merge this, other formatting stuff can be done at a later time.

**Pull request merged**